client/web/src/components/app.tsx

```diff
@@ -18,6 +22,18 @@ export default function App() {
     refreshData()
   }, [auth, refreshData])
 
+  const initialUpdateState =
+    (data?.ClientVersion.RunningLatest) ? UpdateState.UpToDate : UpdateState.Available
+
+  const [updating, setUpdating] = useState<UpdateState>(initialUpdateState)
+
+  const [updateLog, setUpdateLog] = useState<string>('')
+
+  const appendUpdateLog = (msg: string) => {
+    console.log(msg)
+    setUpdateLog(updateLog + msg + '\n')
+  }
```

Comment:
I think we'll want these within `UpdatingView` itself. Avoids having to pass them around between components, and avoids the extra state when not being used.